### ISSUE TITLE
Built-in objects

### ISSUE THREAD

**Issue by VimLeSai** [eslint/eslint#11122]
**Please describe what the rule should do:**

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Enforces code style (layout)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js
      const data = new FormData();
      const data2 = new File();
      const data3 = new FileReader();

in this case eslint will show that FormData, File and FileReader is not defined so eslint should consider this js's built-in objects
```

**Why should this rule be included in ESLint (instead of a plugin)?**
This are the objects that provided by JS, so there should not be warning for this or have to set rule for particuler line or file when it comes to codeing

**Are you willing to submit a pull request to implement this rule?**
i will try

**Comment by ljharb** [comment]
They're not built into JS, they're built into the browser. Try setting `browser: true` in `env`.

**Comment by VimLeSai** [comment]
sorry, i found issue in my code i added ```"env": {
		"browser": true,
		"node": true
	}``` under the rules instead of outside.
